src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.Api/Controllers/StatelessDataController.cs

```diff
@@ -124,14 +178,87 @@ public async Task<ActionResult> Put([FromQuery] string dataType)
                 return BadRequest(deserializer.Error);
             }
 
-            if (appModel == null)
+            // runs prefill from repo configuration if config exists
+            await _prefillService.PrefillDataModel(partyId.ToString(), dataType, appModel);
+
+            Instance virutalInstance = new Instance() { InstanceOwner = new InstanceOwner() { PartyId = partyId.ToString() } };
```

Comment:
Should we include the person number or organization number as well? They can be useful when doing API calls. A real instance would have that populated.